gulpfile.js

```diff
@@ -83,18 +83,10 @@ const qs = () =>
     dest(__dirname + "/ckan/public/base/vendor/")
   )
 
-const highlightJs = () =>
-  src(__dirname + "/node_modules/@highlightjs/cdn-assets/highlight.js").pipe(
-    dest(__dirname + "/ckanext/textview/theme/public/vendor/")
-  )
-
-const highlightJsStyles = () =>
-  src(__dirname + "/node_modules/@highlightjs/cdn-assets/styles/a11y-light.min.css").pipe(
-    rename("a11y-light.css")).pipe(
-    dest(__dirname + "/ckanext/textview/theme/public/styles/")
-  )
-
-
```

Comment:
Initially, gulpfile was used for compiling less/scss into standard CSS. Later we added a hack, that copies updated jquery/higlights/bootstrap/etc. into CKAN vendor folders and that's exactly what you see here.

I'm working on a slightly different approach in #7691. IIRC, these lines that @pdelboca is removing are not working anyway, so there is no harm in removing them. Parts that are affected by these non-working lines will be fixed in #7691

Reply:
PS Moving updated libraries using gulpfile and, on top of this, doing this implicitly, relying on `npm`'s rules that are not very familiar to everyone is probably a wrong decision.
We discussed it briefly with Ian and agreed that using Makefiles for such kinds of tasks may be more efficient. In addition, it will give us a standard tool for adding maintenance commands, like transifex/pypi interaction

Reply:
Using make may not be more efficient but at least one person (me) will be less confused by them :smile: